#global_costmap:
  # Topic to used
#  map_topic: map

  # Set the global and robot frames for the costmap
#  global_frame: map
#  robot_base_frame: base_link

  # Set the update and publish frequency of the costmap
#  update_frequency: 5.0
#  publish_frequency: 0.0

  # We'll use a map served by the map_server to initialize this costmap
#  static_map: true
#  rolling_window: false

global_costmap:
  global_frame: map
  robot_base_frame: base_footprint
  update_frequency: 5.0
  static_map: true

  plugins:
    - {name: static,                  type: "costmap_2d::StaticLayer"}
    - {name: obstacles_layer,         type: "costmap_2d::VoxelLayer"}
    - {name: inflation_layer,         type: "costmap_2d::InflationLayer"}
